lib/wda/xcodebuild.js

```diff
@@ -152,7 +154,6 @@ class XcodeBuild {
     // first do a build phase
     log.debug('Pre-building WDA before launching test');
     this.usePrebuiltWDA = true;
-    this.xcodebuild = await this.createSubProcess(true);
```

Comment:
is it intended?

Reply:
Yes, I noticed while going through the logs to fix a problem, that we created the process here, but then recreated it when we ran start. This one is just thrown away.